package.json

```diff
@@ -59,14 +65,15 @@
     "better-npm-audit": "^3.7.3",
     "cors": "^2.8.5",
     "express": "^4.18.2",
-    "got": "^11.8.5",
+    "got": "^12.5.3",
     "http-errors": "^2.0.0",
-    "lodash": "^4.17.21",
+    "lodash-es": "^4.17.21",
     "memorystream": "^0.3.1",
     "morgan": "^1.10.0",
     "p-filter": "^3.0.0",
     "pump": "^3.0.0",
-    "serverless-http": "^3.1.0",
+    "readable-stream": "^4.3.0",
```

Comment:
there was previously a direct dependency on readable-stream, but it was picked up as a library transitively rather than explicitly. Changing to ESM required it be explicit.